ops/ansible/roles/bfd-db-migrator/templates/bfd-db-migrator-service.sh.j2

```diff
@@ -2,6 +2,7 @@
 export DATABASE_URL='{{ db_migrator_db_url }}'
 export DATABASE_USERNAME='{{ db_migrator_db_username }}'
 export DATABASE_PASSWORD='{{ db_migrator_db_password }}'
+export DATABASE_MAX_POOL_SIZE='{{ db_migrator_db_max_pool_size }}'
```

Comment:
You mentioned that this somehow became required by the java app which seems wrong to me? It seems weird to have to set this. The java code has the reasonable default of 1 already. Maybe there is a bug in the migrator app handling?

Reply:
this is a bug. I left on comment on the other PR just now.